synfig-studio/src/synfigapp/vectorizer/centerlinetostrokes.cpp

```diff
@@ -352,7 +350,7 @@ class SequenceConverter
   double m_penalty;
 
 public:
-  // Length construction globals (see 'lengthOf' method)
+  // Length construction global_variables (see 'lengthOf' method)
```

Comment:
undo change

Reply:
ok I will add that

Reply:
It's still there I guess you missed this. [^_^]